code/game/world.dm

```diff
@@ -29,8 +29,8 @@ GLOBAL_PROTECT(security_mode)
 	LoadBans()
 
 	GLOB.timezoneOffset = text2num(time2text(0,"hh")) * 36000
-
 	Master.Initialize(10, FALSE)
+	initialize_integrated_circuits_list()

```

Comment:
The fuck.

This needs to be a subsystem

Reply:
this is a port, polaris doesnt have the subsystem setup

i'll look into this